source/gui/settingsDialogs.py

```diff
@@ -530,6 +542,14 @@ def makeSettings(self, settingsSizer):
 		self.beepForCapsCheckBox = wx.CheckBox(self, wx.NewId(), label = _("&Beep for capitals"))
 		self.beepForCapsCheckBox.SetValue(config.conf["speech"][getSynth().name]["beepForCapitals"])
 		settingsSizer.Add(self.beepForCapsCheckBox,border=10,flag=wx.BOTTOM)
+		speakDigitsSizer = wx.BoxSizer(wx.HORIZONTAL)
+		# Translators: Label for a combo box for setting how nvda speaks numbers.
+		digitsLabel = wx.StaticText(self, wx.ID_ANY, label=_("Speak &Numbers as"))

```

Comment:
This is the title of an combo box label. Title case isn't used here?


On 9/7/2016 7:12 PM, James Teh wrote:
>
> In source/gui/settingsDialogs.py 
> <https://github.com/nvaccess/nvda/pull/5930#discussion_r77932482>:
>
> > @@ -530,6 +542,14 @@ def makeSettings(self, settingsSizer):
> >  		self.beepForCapsCheckBox = wx.CheckBox(self, wx.NewId(), label = _("&Beep for capitals"))
> >  		self.beepForCapsCheckBox.SetValue(config.conf["speech"][getSynth().name]["beepForCapitals"])
> >  		settingsSizer.Add(self.beepForCapsCheckBox,border=10,flag=wx.BOTTOM)
> > +		speakDigitsSizer = wx.BoxSizer(wx.HORIZONTAL)
> > +		# Translators: Label for a combo box for setting how nvda speaks numbers.
> > +		digitsLabel = wx.StaticText(self, wx.ID_ANY, label=_("Speak &Numbers as"))
>
> nit: Lower case n on numbers. We don't use title case for options.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/nvaccess/nvda/pull/5930/files/13a7a7fdaefae133b7c7e474cba5cb98d4be1b4b#r77932482>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFGivRHn54vlojdNn6Vvi3tcz5wMMquoks5qn2DhgaJpZM4IYz-c>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194